packages/marshal/src/encodePassable.js

```diff
@@ -355,40 +484,45 @@ export const makeEncodePassable = (encodeOptions = {}) => {
         return encodeBigInt(passable);
       }
       case 'remotable': {
-        const result = encodeRemotable(passable, encodePassable);
+        const result = encodeRemotable(passable, innerEncode);
         result.startsWith('r') ||
           Fail`internal: Remotable encoding must start with "r": ${result}`;
         return result;
       }
       case 'error': {
-        const result = encodeError(passable, encodePassable);
+        const result = encodeError(passable, innerEncode);
         result.startsWith('!') ||
           Fail`internal: Error encoding must start with "!": ${result}`;
         return result;
       }
       case 'promise': {
-        const result = encodePromise(passable, encodePassable);
+        const result = encodePromise(passable, innerEncode);
         result.startsWith('?') ||
           Fail`internal: Promise encoding must start with "?": ${result}`;
         return result;
       }
       case 'symbol': {
-        return `y${nameForPassableSymbol(passable)}`;
+        const encName = encodeString(nameForPassableSymbol(passable));
+        return `y${encName.slice(1)}`;
```

Comment:
This part was the holdup today—I didn't realize that symbol encoding essentially duplicates string encoding but with a different prefix and must therefore also be updated for the non-trivial string encoding.

Reply:
assert what initial character it is that you're assuming you're dropping?

Reply:
Looking at this again, I would not add the assertion or comment that I asked for. Fine as is. Resolving.